backend/resources/migrations/postgres/20250829120000_user_display_name.up.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE users ADD COLUMN display_name TEXT;
+
+UPDATE users
+SET display_name = trim(coalesce(first_name,'') || ' ' || coalesce(last_name,''));
+
+ALTER TABLE users ALTER COLUMN display_name SET NOT NULL;
```

Comment:
Two things here:

1. I don't believe updating existing rows is necessary, since if there's no display_name, then we perform this at display time.
2. No need to first add the column, then set it to not null.


```suggestion
ALTER TABLE users ADD COLUMN display_name TEXT NOT NULL;
```